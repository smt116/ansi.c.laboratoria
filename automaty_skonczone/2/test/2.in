    /*/**//*///*//?/*/*//1*/**/2?3?4?//5?*/?6?7?8*//*?**/
    printf("");
    printf("//   ");
    printf(" /* */ ");
    printf("\n");
    printf(" // \n /* \n */ ");
    cout<<" Bla // Bla Bla\n";
    cout<<" Bla /* Bla // */ // Bla"<<endl;
    / ala/ //alka
    /* / ala/ // */
    /ala*/
    /ala*/* ala *./  */
    // komentarz
    // komentarz2 \
    dalszy komentarzyk
    / / ala ma kota
    / * ala ma kota2 */
    perl/*/*/*///*//*//*//**//*//*//*//
    /*/* /* ///*//*//*//**//*//*//*//*/
    komentarz // czy nie komentarz
    *//.*/.**./
    [SPAM]
    /* costam costam costam
    costam costam costam
    costam costam // */
    cout << "przyklad komentarza: /* test
    test test test
    */ ";
    **************/**********************************/***********
    /**/*
    /*cout<<" Bla // Bla Bla\n";*/
    /k\l
    \
    /
    "//"
    "/* */"
    "//'\\/'\/'\/'\/'//\\\/\/''\\/\'/'\\/\\/\/\//\/'/'\\'/'/\'""/'\\/'\/'/'"?/''\"//'\/'\'/\''\\"'/'\'\/"
    puts(" printf(" cout<<" bla "; "); ");
    /*/**//*///*//?/*/*//8*/**/8?8?8?//8?*/?8?8?8*//*?**/