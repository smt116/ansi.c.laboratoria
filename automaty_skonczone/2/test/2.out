Wpisz kod zrodlowy (ctrl+d konczy wpisywanie)

#include "lib.h"


int main() {
    pobieranie_tekstu("Wpisz kod zrodlowy (ctrl+d konczy wpisywanie)\n");

    return 0;
}
